src/defines.h

```diff
@@ -613,7 +606,7 @@
 #define FEAT_MARKER             0xAC /* 172 */
 /* Feature 0xAD -- Underground Tunnel */
 #define FEAT_TAINTED_WATER      0xAE /* 174 */
-#define FEAT_MON_TRAP           0xAF /* 175 */
+/* XXX */
```

Comment:
Don't bother with these XXX markers -- they're just an annoyance and add nothing.